README.md

```diff
@@ -1 +1,76 @@
-# hello_microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+Hello Microverse
+
+> Description the project.
+Set up a linter for HTML and CSS and make sure that any files that are not relevant are ignored by git.
+Added descriptive README file my project.
+Add 3 files each one in a separate commit:
+A .html file that includes a header with the text "Hello Microverse!". Use h1 tag for that.
+A .css file that includes one class with styles for your header (make it in your favorite color).
+A test.md file with the text "This file should be ignored by git".
+
+## Built With
+HTML AND CSS
+
+- Major languages
+- Frameworks
+- Technologies used
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@emekaemmanuel](https://github.com/emekaemmanuel)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/emeka-ugboaja-167820226)
+
```

Comment:
- [OPTIONAL] Please since you are the only author of this project you can singularize the `Authors` section to `Author` and replace `Author1` with your name. :smile: